README.md

```diff
@@ -1025,7 +1025,7 @@ After you have ensured this site is distributing its content legally, you can fo
 5. Add an import in [`youtube_dl/extractor/extractors.py`](https://github.com/rg3/youtube-dl/blob/master/youtube_dl/extractor/extractors.py).
 6. Run `python test/test_download.py TestDownload.test_YourExtractor`. This *should fail* at first, but you can continually re-run it until you're done. If you decide to add more than one test, then rename ``_TEST`` to ``_TESTS`` and make it into a list of dictionaries. The tests will then be named `TestDownload.test_YourExtractor`, `TestDownload.test_YourExtractor_1`, `TestDownload.test_YourExtractor_2`, etc. Note that tests with `only_matching` key in test's dict are not counted in.
 7. Have a look at [`youtube_dl/extractor/common.py`](https://github.com/rg3/youtube-dl/blob/master/youtube_dl/extractor/common.py) for possible helper methods and a [detailed description of what your extractor should and may return](https://github.com/rg3/youtube-dl/blob/7f41a598b3fba1bcab2817de64a08941200aa3c8/youtube_dl/extractor/common.py#L94-L303). Add tests and code for as many as you want.
-8. Make sure your code follows [youtube-dl coding conventions](#youtube-dl-coding-conventions) and check the code with [flake8](https://pypi.python.org/pypi/flake8). Also make sure your code works under all [Python](https://www.python.org/) versions claimed supported by youtube-dl, namely 2.6, 2.7, and 3.2+.
+8. Make sure your code follows [youtube-dl coding conventions](#youtube-dl-coding-conventions) and check the code with [flake8](https://pypi.python.org/pypi/flake8). Example: `flake8 youtube_dl/extractor/example-extractor.py`, or see [flake8 quickstart](http://flake8.pycqa.org/en/latest/index.html#quickstart). Also make sure your code works under all [Python](https://www.python.org/) versions claimed supported by youtube-dl, namely 2.6, 2.7, and 3.2+.
```

Comment:
1. Extractor must be consistently referred as `youtube_dl/extractor/yourextractor.py`.
2. Remove quickstart link and replace pypi one with it instead.

Reply:
updated. actually, I split it to two different points.

btw, what you think of not using inline links, but link references. makes lines a bit more readable in markdown format. currently rather hard to follow in editor.

example:

```diff
-When the tests pass, [add](https://git-scm.com/docs/git-add) the new files and [commit](https://git-scm.com/docs/git-commit) them and [push](https://git-scm.com/docs/git-push) the result, like this

+When the tests pass, [add][git-add] the new files and [commit][git-commit] them and [push][git-push] the result, like this
+
+[git-add]: https://git-scm.com/docs/git-add
+[git-commit]: https://git-scm.com/docs/git-commit
+[git-push]: https://git-scm.com/docs/git-push
```

extra benefit: the ref-links are all in place and easy to update with new release links.

Reply:
References can be useful for DRYing URLs appearing multiple times (e.g. link to info dict meta fields) in document. For single ones I don't see much point using them.